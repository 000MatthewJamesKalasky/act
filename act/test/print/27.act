defproc test()
{
  int x[4];

  chp {

    x[0] := x[0] + 2;(,i:0: x[i] := x[i] + 1 ); x[1] := x[1] + 4
  }
}

test t;
