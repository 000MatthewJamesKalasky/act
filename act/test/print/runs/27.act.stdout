defproc test ();

defproc test ()
{

/* instances */
int<32> x[4];

/* connections */
chp {
x[0]:=x[0]+0x2;skip;x[1]:=x[1]+0x4
}
}


/* instances */
test t;

/* connections */
